assets/schemas/manifest.json

```diff
@@ -1490,16 +1533,14 @@
       "items": {
         "type": "object",
         "properties": {
+          "id": "string",
```

Comment:
It doesn't work this way. You need to say
```
{
  "type": "string"
}
```

Reply:
What is `id` anyway?

Reply:
legalBasisRestrictions in a purpose is an array of strings each of which is presumably a reference to a legalBasisRestriction object. Did you have something different in mind here? Not sure what natural value of id is - not super clean. Could add directly in purpose. Thoughts?

Reply:
I don't think we use ID's anywhere, especially when manually referencing things.

Reply:
Changed to code. Otherwise can move inside of purposes / purposeTemplates. Did not because may be reused across purposeTemplates